views/forms/ConfigForm/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Button } from '@material-ui/core'
+import ContentAdd from '@material-ui/icons/Add'
+import Save from '@material-ui/icons/Save'
+import ConfigFormFields from '../ConfigFields'
+import Form from '../Form'
+
+const ConfigForm = ({ onSubmit, classes, onAddNewField, ...rest }) => {
+  return (
+    <Form onSubmit={onSubmit}>
+      <ConfigFormFields classes={classes} {...rest} />
+      <div className={classes.configFormButtonContainer}>
+        <Button
+          variant="fab"
+          className={classes.addNewFieldButton}
+          iconStyle={{
+            fill: 'rgb(255, 64, 129)',
```

Comment:
Does the `fab` variant always have this iconStyle?

Reply:
good catch. It doesn't removed [99429b6](https://github.com/AMP-SCZ/dpdash/pull/332/commits/99429b662ad457f431dc42e058b5caf3a9747688)